contentpacks/khanacademy.py

```diff
@@ -579,6 +582,72 @@ def retrieve_kalite_data(lang="en", force=False) -> list:
     with open(node_data_path, 'r') as f:
         node_data = ujson.load(f)
 
+    if not lang == "en" and not no_dubbed_videos == True:
```

Comment:
That if statement means if language not equal to "en" and not a no_dubbed_videos it will execute the following:

- Generate `en_nodes.json. file.
- Generate `dubbed_videos_mappings.json file.
- Match youtube ids form en_nodes to dubbed_videos_mappings and append it to node_data.

Else it will not execute the list above.